### PR TITLE
feat: [Updated] Loading screen listener with percent and message

### LINKED ISSUE #1556: Add "qr_loading" event to be triggered after the QR is scanned
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

After scanning the QR, I'm unsure if the scan was successful or not.

### Describe the solution you'd like.

Currently after the QR is scanned, there is no event indicating that Whatsapp Web is loading the session, even though there are visual indications of it.

Would be nice if we could have some sort of `qr_loading` event, that triggers after the QR is scanned, to indicate it was successfully scanned. 

### Describe an alternate solution.

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by tonbotfy**
It will help a lot of people who need this whatsapp screen status.

When the screen is loading, the 'loading_screen' event is called with two parameters:
percent
message
174875294-c06e1263-a5c8-4183-96a4-0fb6f25f8177

Updated from PR #1533 
Referenced by Issue #1556

### CODE REVIEW

**Comment by tonbotfy** [comment]
> 

Hi! Yes it shows the real % of the whatsapp page. I could already see yes, it depends a lot on the cash of your machine with your cell phone.

**Review by purpshell — APPROVED** [review]
looks really good to me, thanks for adding this

**Review by purpshell — CHANGES_REQUESTED** [review]
fix eslint errors then I'll re-approve it

**Comment by tonbotfy** [comment]
@PurpShell 
> fix eslint errors then I'll re-approve it

Done! 🥰

File: example.js [hunk 1]
```diff
@@ -2,11 +2,15 @@ const { Client, Location, List, Buttons, LocalAuth} = require('./index');
 
 const client = new Client({
     authStrategy: new LocalAuth(),
-    puppeteer: { headless: false }
+    puppeteer: { headless: true }
```
Comment (jtourisNS) [inline example.js:5]: Just curious, why did you change this?

**Review by jtouris — APPROVED** [review]

**Review by purpshell — APPROVED** [review]
Looks good

**Comment by tonbotfy** [comment]
@pedroslopez can you marge/approve this pr?

File: example.js (hunk 1 above)
Comment (jtourisNS) [reply example.js:5]: @tonbotfy

Comment (tonbotfy) [reply example.js]: @jtourisNS Done, my friend! 👍🏼

File: index.d.ts [hunk 2]
```diff
@@ -241,6 +241,9 @@ declare namespace WAWebJS {
             message: Message
         ) => void): this
 
+        /** Emitted when loading screen is appear */
```
Comment (stefanfuchs) [inline index.d.ts:244]:
```suggestion
        /** Emitted when loading screen is appearing */
```

**Review by jtouris — CHANGES_REQUESTED** [review]
Remember to also add the `LOADING_SCREEN` event on the Client enum Events - `index.d.ts #455`

**Comment by tonbotfy** [comment]
> Remember to also add the `LOADING_SCREEN` event on the Client enum Events - `index.d.ts #455`

Done! 👍🏼

**Comment by tonbotfy** [comment]
Any answer?

**Review by tuyuribr — APPROVED** [review]
LGTM! 
WAWEB: 2.2228.14

**Comment by tonbotfy** [comment]
@tuyuribr Thanks. let's go to approval! 😍😍

**Review by janke184 — APPROVED** [review]

**Pull request merged**